src/lib/autoinstall/autoinst_issues/issue.rb

```diff
@@ -0,0 +1,52 @@
+module Y2Autoinstallation
+  module AutoinstIssues
+    # Base class for autoinstallation problems while importing the
+    # AutoYaST configuration file.
+    #
+    # Y2Autoinstallation::AutoinstIssues offers an API to register
+    # and report related AutoYaST problems.
+    class Issue
+      include Yast::I18n
+
+      # @return [#parent,#section_name] Section where it was detected
```

Comment:
This one should be an `String`, right? (it was an slightly different thing in `yast2-storage-ng`).